public/locale/en.json

```diff
@@ -275,6 +275,7 @@
   "aadhaar_otp_send_success": "OTP has been sent to the mobile number registered with the Aadhar number.",
   "aadhaar_validation_length_error": "Should be a 12-digit aadhaar number or 16-digit virtual ID",
   "aadhaar_validation_space_error": "Aadhaar number should not contain spaces",
+  "abort": "Abort",
```

Comment:
reuse cancel instead

```suggestion
```

Reply:
@YourBroCode  ☝️ Use cancel instead of abort, make sure to remove abort from en.json as well.